content-src/selectors/selectors.js

```diff
@@ -7,26 +7,59 @@ const urlParse = require("url-parse");
 const SPOTLIGHT_LENGTH = module.exports.SPOTLIGHT_LENGTH = 3;
 const TOP_SITES_LENGTH = module.exports.TOP_SITES_LENGTH = 6;
 const BACKGROUND_FADE = 0.5;
-const DEFAULT_FAVICON_BG_COLOR = [255, 255, 255];
+const DEFAULT_FAVICON_BG_COLOR = [150, 150, 150];
```

Comment:
Ha! Didn't we just change this to white 2 days ago in https://github.com/mozilla/activity-streams/pull/511/commits/4fec351d2e4cdd6ff69980820b5c380548742bef?